### ISSUE TITLE
Test if AssetGraph will run and build under Ubuntu for Windows.

### ISSUE THREAD

**Issue by gustavnikolaj** [assetgraph/assetgraph#472]
http://blog.dustinkirkland.com/2016/03/ubuntu-on-windows.html?m=1

/cc @Munter @papandreou 

I'll take a look when it gets released to the general public.

**Comment by mklabs** [comment]
I can maybe help with the testing. I'm running Bash on Windows as well (actually fish on bash on ubuntu on windows: https://gist.github.com/mklabs/3124b0b8fa94b69eacf8d38c467c65c9)

**Comment by gustavnikolaj** [comment]
Sure! Let us know how it works out for you :-)

**Comment by Munter** [comment]
I guess a clone, `npm install` and `npm test` would be enough to answer that question. The real challenge is probably going to be assetgraph-builder with all the external dependencies

**Comment by gustavnikolaj** [comment]
According to
http://www.hanselman.com/blog/DevelopersCanRunBashShellAndUsermodeUbuntuLinuxBinariesOnWindows10.aspx
apt packages should be available.

But given the amount of dependencies I guess we should expect to hit some
problems.

Den tirsdag den 7. juni 2016 skrev Peter Müller <notifications@github.com>:

**Comment by gustavnikolaj** [comment]
I checked out assetgraph on Bash on Ubuntu on Windows with node 7.10 installed. Both npm install and npm test worked.

I also tested assetgraph-builder. npm install worked when I installed the necessary apt dependencies. npm test works too.

Seems that it is quite slow though. The npm install performance was really bad and the assetgraph test suite spent 45 seconds running where the builder test suite spent about 2 minutes.

Overall, it seems that we can claim support of the windows linux sub system without paying too much attention to it :-)
